src/bam-utils.js

```diff
@@ -94,7 +94,6 @@ export const parseMD = (mdString, useCounts) => {
    */
 export const getSubstitutions = (segment, seq) => {
   let substitutions = [];
-  let insertions = 0;
```

Comment:
This is unused except for the increment a few lines later.

Reply:
Good catch. Thanks.